bert/src/glue/finetuning_jobs.py

```diff
@@ -0,0 +1,749 @@
+# # Copyright 2022 MosaicML Composer authors
+# # SPDX-License-Identifier: Apache-2.0
+
+# """Contains GLUE job objects for the simple_glue_trainer."""
+import multiprocessing as mp
+from typing import Any, Dict, List, Optional, Union, cast
+
+import torch
+from torch.utils.data import DataLoader
+
+from composer.core import Callback
+from composer.core.evaluator import Evaluator
+from composer.core.types import Dataset
+from composer.loggers import LoggerDestination
+from composer.optim import ComposerScheduler, DecoupledAdamW
+try:
```

Comment:
Adding this comment for myself:

This try/except can be removed once composer/v0.11.1 is released and this entrypoint is tied to that release.

The new code can simply be:
```from composer.trainer.devices import Device, DeviceGPU```